### PR TITLE
Bash short cut

### PR DESCRIPTION
**Pull request by Bowenislandsong**
The current status is able to create and destroy windows node and security group for RDP.
All requirements are met and code is locally tested
This project uses go.mod
simple command including 
```bash 
./winc_node_setup create -awscred=/abs/path/to/your/aws/credentials -kubeconfig=/abs/path/to/your/kubeconfig

./winc_node_setup destroy -awscred=/abs/path/to/your/aws/credentials
```
/cc @gmarkley-VI
/cc @ravisantoshgudimetla
/cc @aravindhp

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,16 +1,62 @@
...
 	}
-	ec2_instances.CreateEC2WinC(sessAWS, oc, "", "", "")
+	createFlag := flag.NewFlagSet("create", flag.ExitOnError)
+	destroyFlag := flag.NewFlagSet("destroy", flag.ExitOnError)
+	if os.Args[1] == "create" {
+		// subflags of create
+		// openshift cluster region
+		credPath := createFlag.String("awscred", "", "Required: absolute path of aws credentials")
+		credAccount := createFlag.String("awsaccount", "openshift-dev", "account name of the aws credentials") // Default accounts for dev team in OpenShift
+		dir := createFlag.String("dir", "./", "path to 'winc-setup.json'.")
```
Comment (gmarkley-VI) [inline main.go]: Would it make more sense to be able to feed this in the same way we do for the installer that way we keep the config inche dir with the info for the deployment?

Comment (Bowenislandsong) [reply main.go]: That’s a great idea, and also what I was hoping for as well. However, there is no information telling me the path where Openshift-installer dir is at. We could argue that most likely kubeconfig is in the directory’s auth folder, but we can’t be sure. Hence here we ask as a flag to specify ( I just noticed that we might need to parse the dir than handle as a string as I am doing it now. I will change it in a bit. ) the default value is the current directory just like the installer

**Comment by gmarkley-VI** [comment]
Just a flag and we can specify would work just like the installer no need to auto find it.

**Comment by Bowenislandsong** [comment]
I’m on my phone, I’m not sure if I’m seen this correctly, but the newest version (from a day a ago) should be a flag. dir := createFlag.String("dir", "./", "path to 'winc-setup.json'.")

**Comment by gmarkley-VI** [comment]
You are Correct. Looks great. I am on phone too sorry for any confusion

**Comment by Bowenislandsong** [comment]
That’s ok, thanks for the input. I like that we think similarly in designing flags!

**Pull request merged**